Source/Public/Publish-Bicep.ps1

```diff
@@ -1,14 +1,30 @@
 function Publish-Bicep {
     [CmdletBinding()]
     param (
-        [Parameter(Mandatory, Position = 1)]
+        [Parameter(ParameterSetName = 'Default', Mandatory, Position = 1)]
+        [Parameter(ParameterSetName = 'Registry', Position = 1)]
         [ValidateNotNullOrEmpty()]
-        [ValidateScript({Test-Path $_})]
+        [ValidateScript( { Test-Path $_ })]
         [string]$Path,
 
-        [Parameter(Mandatory, Position = 2)]
+        [Parameter(ParameterSetName = 'Default', Position = 2)]
         [ValidateNotNullOrEmpty()]
-        [string]$Target          
+        [string]$Target,
+        
+        [Parameter(ParameterSetName = 'Registry', Mandatory, Position = 2)]
+        [ValidateNotNullOrEmpty()]
+        [string]$Registry,
```

Comment:
should $registry input include the 'br:' or 'br/'? If it should, then we need to clean the url here too before constructing it on line 55 / 58, if you should never input the br part, again, we need validation.